docs/changes/0.25.0.rst

```diff
@@ -9,28 +9,63 @@ Breaking Changes:
 
 - `get_data_by_id` no longer returns data for standalone parameters
   if there are no dependent parameters in the dataset. For the same reason
-  `plot_by_id` and `plot_dataset` will no longer plot these standalone parameters.
-- `xarray` exported data no longer exports extra metadata columns under the `extra_metadata` tag
-  since the extra dictionary level prevents exporting to `netcdf`
-
+  `plot_by_id` and `plot_dataset` will no longer plot these standalone
+  parameters. (#2918)
+- `xarray` exported data no longer exports extra metadata columns under the
+  `extra_metadata` tag since the extra dictionary level prevents exporting to
+  `netcdf` (#2934)
 
 ------------
 Deprecation:
 ------------
 
 - `get_data_by_id` and `get_shaped_data_by_runid` are now deprecated. These
   functions now all have better alternatives in the build in `get_parameter_data` as well as `Xarray`
-  and `Pandas` exporters.
+  and `Pandas` exporters. (#2921, #2925)
 
 ----
 New:
 ----
 
+- SP983c Basel IV amplifier remote control driver (#2707)
+- Add support for Keysight 34411A (#2927)
+- Added the functionality of getting guids from multiple run ids and directly
+  from database (#2930)
+- NumpyJSONEncoder: Add support for the uncertanties' UFloat type (#2933)
+- Add conversion between qcodes legacy DataSet and xarray format (#2938)
+
 ---------
 Improved:
 ---------
 
+- plot_dataset avoid loading from db if in cache (#2918)
```

Comment:
@jenshnielsen could you help with a user-friendly formulation of this?
```suggestion
- plot_dataset: improve performance by avoiding loading data from the database
  if the data is already in dataset's cache (#2918)
```

Reply:
Perhaps:

```plot_dataset: Improved performance by using in memory cache when possible to avoid reloading data from disk```

Reply:
agree! :)

Reply:
updated :)